components/display_drivers/include/sh8601.hpp

```diff
@@ -0,0 +1,364 @@
+#pragma once
+
+#include <mutex>
+
+#include "display_drivers.hpp"
+
+namespace espp {
+/**
+ * @brief Display driver for the GC9A01 display controller.
```

Comment:
Please update the doxy comment for this class when you're done to have correct info about the driver :)